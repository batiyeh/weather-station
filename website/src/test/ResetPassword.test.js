import React from 'react'
import ResetPasswordForm from '../components/registration/ResetPasswordForm'
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
    .create(<ResetPasswordForm/>).toJSON();
    expect(tree).toMatchSnapshot();
})