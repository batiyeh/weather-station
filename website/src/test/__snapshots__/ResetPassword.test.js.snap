// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="forgot-container"
>
  <h2
    className="login-title"
  >
    Reset Password
  </h2>
  <form
    action="/api/user/reset/"
    className="ResetPasswordForm"
    method="post"
  >
    <div
      className="form-group"
    >
      <input
        class="form-control"
        id="email"
        name="email"
        placeholder="Email"
        type="email"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="col-6 no-padding-left"
      >
        <a
          className="register-link"
          href="/user/login"
        >
          <button
            className="btn btn-default btn-block register-btn"
            type="button"
          >
            Return
          </button>
        </a>
      </div>
      <div
        className="col-6 no-padding-right"
      >
        <button
          className="btn btn-default btn-block login-btn"
          type="submit"
        >
          Recover Password
        </button>
      </div>
    </div>
  </form>
</div>
`;
